src/pages/faq.tsx

```diff
@@ -0,0 +1,64 @@
+import { Collapse, Text } from '@geist-ui/core';
+import { useRouter } from 'next/router';
+import { ReactMarkdown } from 'react-markdown/lib/react-markdown';
+
+const FAQ = () => {
+  const FAQ = [
+    {
+      Name: 'AvdanOS',
+      Questions: [
+        {
+          QuestionNumber: 1,
+          Title: 'What will AvdanOS be?',
+          SubTitle: '',
+          Answer:
+            'We are simply doing a Desktop Environment. But don’t get too mad, we don’t need more than that. We do not need to code or package a whole OS.\n\n> **Linux package =** “package” refers to a **compressed file archive containing all of the files that come with a particular application**. The files are usually stored in the package according to their relative installation paths on your system. Most packages also contain installation instructions for the OS, as well as a list of any other packages that are dependencies (prerequisites required for installation.\n> \n\n🧠 This means that we will need to simply just make a desktop environment and package it. Linux will figure it out.\n\nThere is a **huge misconception that the UI you see when you use a particular OS is the OS itself**. It is not. Those are 2 completely different things. The **OS** is what *screws around with all the hardware* you have (i.e. the Kernel and the System Programs). The **applications** (Application Programs to be precise) that you use, including the desktop UI *is not a part of the operating system*.\n\n🧠 We will also write a **Wayland Compositor [The Wayland Compositor ](https://www.notion.so/The-Wayland-Compositor-d50267f916ae4592bfe9a3340e5472f1)** to allow for the custom features and behaviour we want to take place. It will be made using **Rust** and [**Smithay**](https://smithay.github.io/) to add additional Features to the Desktop Environment.\n'
```

Comment:
Consider using single quote instead of U+2019 and wrapped in double quotes. It's used more often. Do you not get this warning in VSCode? 

<img width="464" alt="Screenshot 2022-08-08 at 12 14 11 PM" src="https://user-images.githubusercontent.com/51555391/183337792-7d1e1f43-dbcb-4813-b052-9298b675f357.png">

Reply:
Its not written by me, i just copy and pasted from notion

Reply:
You can edit it afterwards when all questions will be added